### DISCUSSION TITLE
When are the entity embedding vector data saved?

### DISCUSSION THREAD

**Discussion by travisgu** [microsoft/graphrag#1274] [Q&A]
After I executed the graphrag.index and graphrag.query in command line, I didn't find any embedding vector saved in the parquet file. And the lance db seems only stores the entity description text embedding.
I checked the entity.py and there is description_embedding/name_embedding/graph_embedding fields. In the graphrag/query/context_builder/entity_extraction.py it is calling the vector store to search to use the above embeddings. 

Can anyone kindly help to point out where name_embedding/graph_embedding vector data are saved? 

PS: in the settings.yaml file I already set the embed_graph "enabed" to true.

**Comment by lawyinking** [comment]
you need to set:

```
embed_graph:
  enabled: true # if true, will generate node2vec embeddings for nodes
```

**Comment by satyaloka93** [comment]
Just started experimenting with graphrag, after creating the parquet files I also didn't get embeddings saved. How do you run graphrag just to create embeddings after already making parquets?

**Reply by mmaitre314** [reply]
In our case, we had to set `snapshots.embeddings: true` in the YAML config to get embeddings saved as separate Parquet files.
